frontend/src/app/components/carousel/carousel.component.css

```
@@ -0,0 +1,19 @@
+.carousel {
+    margin: auto;    
+    width: 90%;
+}
+
+.carousel-inner {
+    border-radius: 6px;
+}
+
+.carousel-item {
+    height: 300px;
+    background-color: black;
+}
+
+.carousel-item img {
+    height: 100%;
+    width: 100%;
+    object-fit: contain;
+}
```

- Não sei o motivo, mas usar a classe `carousel-item` com o `ngb-carousel` não funcionou, o estilo não é aplicado. Por causa disso, usei apenas a classe `carousel-inner`.
- O `border-radius` foi removido daqui por que o Bootstrap já tem um estilo pré-definido pra isso (`rounded`).
```suggestion
.carousel-inner {
    background-color: black;
    height: 300px;
}

.carousel-inner img {
    width: 100%;
    height: 100%;
    object-fit: contain;
}
```